docs/fides/docs/tutorial/add.md

```diff
@@ -16,22 +16,17 @@ Then, install the dependencies by running:
 pip install -r requirements.txt
 ```
 
-## Configure Fidesctl
+## Initializing Fidesctl
 
-Fidesctl needs some configuration to work for your environment; this is handled by looking for a TOML file in the current working directory. Create a `fidesctl.toml` file at the root level of the repository. It should contain the following configuration:
+With fidesctl installed, it's time to initialize the project so we have some place to start adding resource manifests and tweaking our configuration. Run the following command and follow the prompts to get your local fidesctl instance initialized.
```

Comment:
Should there not be any other prompts at this point? `fidesctl init` creates the directory and a config file then exits for me

Reply:
yep! this is all that it does currently, but we will be adding more as we go